packages/expo-calendar/src/Calendar.ts

```diff
@@ -757,19 +757,17 @@ export async function createEventAsync(
     throw new Error('createEventAsync must be called with an id (string) of the target calendar');
   }
 
-  const { id, ...details } = eventData;
```

Comment:
Let's leave that logic for now. Some users don't use typescript, so they won't know about this breaking change. However, we should add a runtime warning if the `id` is defined. What do you think about it?

Reply:
Oh right good catch I hadn't thought about that, I'll make the change

Reply:
I put back the logic. I had to use a ts-expect-error though because TS complained `id` wasn't defined